app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -611,11 +611,19 @@ class MultipleDisposalsTriageController @Inject() (
     withMultipleDisposalTriageAnswers(request) { (_, state, triageAnswers) =>
       val isIndividual = state.fold(_.subscribedDetails, _._1.subscribedDetails).userType().isRight
 
+      val representeeAnswers = state
+        .fold(
+          _.representeeAnswers,
+          _._2.representeeAnswers
+        )
+      val isIncomplete = !representeeAnswers.map(_.fold(_ => false, _ => true)).getOrElse(false)
```

Comment:
`isIncomplete` is too general - something like `representeeAnswersIncomplete` is more precise